src/main/java/dev/jbang/Main.java

```diff
@@ -6,12 +6,16 @@
 import dev.jbang.cli.JBang;
 
 import picocli.CommandLine;
+import picocli.jansi.graalvm.AnsiConsole;
 
 public class Main {
 	public static void main(String... args) {
-		CommandLine cli = JBang.getCommandLine();
-		args = handleDefaultRun(cli.getCommandSpec(), args);
-		int exitcode = cli.execute(args);
+		int exitcode;
+		try (AnsiConsole ansi = AnsiConsole.windowsInstall()) {
```

Comment:
should it not just fail silently if install does not work?

Reply:
The install is just the resource here which is being closed again. Maybe a bit overkill with System.exit() as the very next thing, but this way it won't get lost if refactored.